components/lib/crash/metrics.yaml

```diff
@@ -0,0 +1,27 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# This file defines the metrics that are recorded by glean telemetry. They are
+# automatically converted to Kotlin code at build time using the `glean_parser`
+# PyPI package.
+
+$schema: moz://mozilla.org/schemas/glean/metrics/1-0-0
+
+crash_metrics:
+  crash_count:
+    type: labeled_counter
+    description: >
+      Counts the number of crashes that occur in the application.  This measures only the counts of
+      each crash in association with the labeled type of the crash. The labels correspond to the
+      types of crashes handled by lib-crash.
+    labels:
+      - uncaught_exception
+      - native_code_crash
+    bugs:
+      - 1553935
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1553935#c3
```

Comment:
I noticed that the data-review on this is still pending, could you check in with @liuche about it?
I suspect that application using lib-crash + Glean are still required to ask for data review. Let's also clear this with @liuche before merging because, if it does, we need to explicitly state that in the README file.

Reply:
Note added to README.md